front/lib/api/assistant/conversation/title.ts

```diff
@@ -48,6 +50,24 @@ export async function ensureConversationTitle(
   }
   const auth = await Authenticator.fromJSON(authType);
 
+  // If the last message is a function call without tool output,
+  // We strip it for title generation otherwise the model will throw.
+  const conversationContent = clone(conversation.content);
+  const lastConversationBatch =
+    conversationContent[conversationContent.length - 1];
+  const lastConversationBatchMessage =
+    lastConversationBatch[lastConversationBatch.length - 1];
+  if (lastConversationBatchMessage?.type === "agent_message") {
+    while (
+      lastConversationBatchMessage.contents[
+        lastConversationBatchMessage.contents.length - 1
+      ]?.content.type === "function_call"
+    ) {
+      lastConversationBatchMessage.contents =
+        lastConversationBatchMessage.contents.slice(0, -1);
+    }
+  }
+
   const titleRes = await generateConversationTitle(auth, {
     ...conversation,
     content: [...conversation.content, [userMessage]],
```

Comment:
Should it use the conversationContent you cloned here ?
```suggestion
    content: [...conversationContent [userMessage]],
```

Reply:
Indeed, thanks !